devices/ds18b20/ds18b20.go

```diff
@@ -24,10 +24,13 @@ package ds18b20
 import (
 	"errors"
 	"fmt"
+	"strconv"
+	"strings"
 	"time"
 
 	"periph.io/x/periph/conn/onewire"
 	"periph.io/x/periph/devices"
+	"periph.io/x/periph/host/sysfs"
```

Comment:
Normally the packages in /devices should not depend on /host, but /conn instead. The current exception is /host/cpu for https://periph.io/x/periph/host/cpu#Nanospin